### PR TITLE
Add usage documentation for `ns_allow_kill_commands`

### LINKED ISSUE #90: Document `ns_allow_kill_commands`
https://github.com/R2Northstar/NorthstarMods/pull/414/files#diff-5195b7170a1c3fdba5c176fc2bef41d2917dc590c6291cf95a02e4f44df67ee5R54

### PR DESCRIPTION
**Pull request by lizardwithawizard**
Added `ns_allow_kill_commands` to the convars section of the hosting guide

Closes #90

### CODE REVIEW

File: docs/hosting-a-server-with-northstar/dedicated-server/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Example: `+setplaylistvaroverrides "run_epilogue 0 featured_mode_amped_tacticals
 | `oob_timer_enabled`                          | `0-1`           |               | Out of bounds timer enable                                                                      |
 | `roundtimelimit`                             |                 |               |                                                                                                |
 | `classic_rodeo`                              | `0-1`           |               |                                                                                                |
-| `classic_mp`                                 | `0-1`           | `1`           | Enables intro dropship                                                                         |
+| `classic_mp`                                 | `0-1`           | `1`           | Enables intro dropship                                                                          |
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/dedicated-server/README.md]:
```suggestion
| `classic_mp`                                 | `0-1`           | `1`           | Enables intro dropship                                                                         |
```

Unrelated space added

File: docs/hosting-a-server-with-northstar/dedicated-server/README.md [hunk 2]
```diff
@@ -130,6 +130,7 @@ They allow the server admin to set server's properties like the name, TCP port,
 | `ns_private_match_last_mode`                     | Forces the lobby to a specific Gamemode                                                                                                                                                     | `tdm`                          | Any [Gamemode](./#gamemodes)                       |
 | `ns_private_match_last_map`                      | Forces the lobby to a specific map                                                                                                                                                          | `mp_forwardbase_kodai`         | Any [Map](./#maps)                                 |
 | `ns_disallowed_weapons`                          | Blacklists weapons                                                                                                                                                                          |                                | List of [Weapons](./#weapons) separated by a comma |
+| `ns_allow_kill_commands`                     | Enables the use of `kill` in the console           | `0`           | `0-1`                                                        |
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/dedicated-server/README.md]: Any chance you could get the indention to match up?
(Kinda a pain to do via GitHub suggestions)

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
Thanks for the addition. Just need to fix formatting and then it should be good to merge ^^

File: docs/hosting-a-server-with-northstar/dedicated-server/README.md (hunk 2 above)
Comment (lizardwizardonthewall) [reply docs/hosting-a-server-with-northstar/dedicated-server/README.md]: yeah, I'll work on this rq

Comment (lizardwizardonthewall) [reply docs/hosting-a-server-with-northstar/dedicated-server/README.md]: hopefully thats better, it matches up on my end

Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/dedicated-server/README.md]: Still scuffed for me, trying to fix it on my end now ^^

File: docs/hosting-a-server-with-northstar/dedicated-server/README.md [hunk 3]
```diff
@@ -130,6 +130,7 @@ They allow the server admin to set server's properties like the name, TCP port,
 | `ns_private_match_last_mode`                     | Forces the lobby to a specific Gamemode                                                                                                                                                     | `tdm`                          | Any [Gamemode](./#gamemodes)                       |
 | `ns_private_match_last_map`                      | Forces the lobby to a specific map                                                                                                                                                          | `mp_forwardbase_kodai`         | Any [Map](./#maps)                                 |
 | `ns_disallowed_weapons`                          | Blacklists weapons                                                                                                                                                                          |                                | List of [Weapons](./#weapons) separated by a comma |
+| `ns_allow_kill_commands`                         | Enables the use of `kill` in the console  | `0`           | `0-1`                                                 |
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/dedicated-server/README.md]:
```suggestion
| `ns_allow_kill_commands`                         | Enables the use of `kill` in the console                                                                                                                                                    | `0`                            | `0-1`                                              |
```

Format fix attempt

File: docs/hosting-a-server-with-northstar/dedicated-server/README.md (hunk 2 above)
Comment (GeckoEidechse) [reply docs/hosting-a-server-with-northstar/dedicated-server/README.md]: Managed to fix in 2bcc6096c0929cfcbd370b538fcbe6ebe4de9055 ^^

**Review by GeckoEidechse — APPROVED** [review]
Looks good now ^^

**Pull request merged**